src/core/server/saved_objects/saved_objects_config.ts

```diff
@@ -18,7 +18,7 @@ export const savedObjectsMigrationConfig = {
     pollInterval: schema.number({ defaultValue: 1500 }),
     skip: schema.boolean({ defaultValue: false }),
     // TODO migrationsV2: remove/deprecate once we release migrations v2
-    enableV2: schema.boolean({ defaultValue: false }),
+    enableV2: schema.boolean({ defaultValue: true }),
```

Comment:
Should the comment be updated?

Reply:
I'll deprecate the setting in https://github.com/elastic/kibana/pull/85778/files when we know all tests pass with the new migration.